src/store/ducks/tests/fixtures.ts

```diff
@@ -288,8 +288,8 @@ export const planRecordsById = {
     plan_date: '2019-05-24',
     plan_effective_period_end: '2019-09-30',
     plan_effective_period_start: '2019-05-27',
-    plan_fi_reason: '',
-    plan_fi_status: '',
+    plan_fi_reason: ' NULL',
```

Comment:
Why are we doing this? 

- is it possible to get plans with the Fi reason and status as `''`?
- is it possible to get plans with the Fi reason and status as `' NULL'`?